runners/spark/src/main/java/org/apache/beam/runners/spark/translation/BoundedDataset.java

```diff
@@ -46,6 +46,7 @@
   private Iterable<WindowedValue<T>> windowedValues;
   private Coder<T> coder;
   private JavaRDD<WindowedValue<T>> rdd;
+  private List<byte[]> clientBytes;
```

Comment:
kinda have issues with that name but cannot think at the moment on something better apart of the even worse `bytes`